src/osd/PG.cc

```diff
@@ -453,6 +454,20 @@ unsigned PG::get_scrub_priority()
   return pool_scrub_priority > 0 ? pool_scrub_priority : cct->_conf->osd_scrub_priority;
 }
 
+unsigned PG::get_scrub_cost()
+{
+  auto num_bytes = static_cast<uint64_t>(
+    std::max<int64_t>(
+      0, 
+      info.stats.stats.sum.num_bytes));
+  auto num_objects = static_cast<uint64_t>(
+    std::max<int64_t>(
+      1, 
+      info.stats.stats.sum.num_objects));
+  uint64_t cost_per_object = std::max<uint64_t>(num_bytes / num_objects, 1);
```

Comment:
This duplicates logic in queue_recovery() -- factor it out into a helper (object_stat_sum_t::get_average_object_size() const)